prov/ansible/execution.go

```diff
@@ -895,25 +895,26 @@ func (e *executionCommon) executeWithCurrentInstance(ctx context.Context, retry
 		return err
 	}
 
-	ansibleRecipePath := filepath.Join(ansiblePath, stringutil.UniqueTimestampedName(e.taskID+"_", ""), e.NodeName)
+	ansibleExecutionRootDir := filepath.Join(ansiblePath, stringutil.UniqueTimestampedName(e.taskID+"_", ""))
+	ansibleRecipePath := filepath.Join(ansibleExecutionRootDir, e.NodeName)
 	if e.operation.RelOp.IsRelationshipOperation {
 		ansibleRecipePath = filepath.Join(ansibleRecipePath, e.relationshipType, e.operation.RelOp.TargetRelationship, e.operation.Name, currentInstance)
 	} else {
 		ansibleRecipePath = filepath.Join(ansibleRecipePath, e.operation.Name, currentInstance)
 	}
 
-	if err = os.RemoveAll(ansibleRecipePath); err != nil {
-		err = errors.Wrapf(err, "Failed to remove ansible recipe directory %q for node %q operation %q", ansibleRecipePath, e.NodeName, e.operation.Name)
+	if err = os.RemoveAll(ansibleExecutionRootDir); err != nil {
```

Comment:
Is this still relevant?
If we generate a unique timestamped name..

Reply:
yes I didn't remove it because it doesn't hurt, but indeed it is not needed, and was not needed before this commit either since the directory was already timestamped.
Will remove it.